src/layout/AuthLayout.tsx

```diff
@@ -0,0 +1,17 @@
+import { Layout } from 'react-admin'
+import { AuthAppBar } from './AuthAppBar'
+
+
+// Added children to display the body
+const AuthLayout = ({ props, children }) => (
```

Comment:
I believe this won't work as there is not field `props` passed
```suggestion
const AuthLayout = ({ children, ...props }) => (
```